### PR TITLE
Add LeNet example based nn v2 API

### LINKED ISSUE #1301: [Polyfill][Example] add LeNet-based handwritten digits classification example
The related webnn issue: https://github.com/webmachinelearning/webnn/issues/70

### PR DESCRIPTION
**Pull request by huningxin**
fix #1301

### CODE REVIEW

**Comment by huningxin** [comment]
@BruceDai @ibelem , PTAL. Thanks!

File: examples/lenet/index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        <div class="col-4"></div>
+        <div class="col">
+        </div>
+      </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js"></script>
+    <script src="../../dist/webml-polyfill.js"></script>
+    <script src="lib/mnist.js"></script>
```
Comment (BruceDai) [inline examples/lenet/index.html:104]: @huningxin Now **mnist.js** is under mnist folder, you maybe forgot to update this src path after you renamed lib to mnist, please update 
```js
src="lib/mnist.js"
```
to
```js
src="mnist/mnist.js"
```

Comment (huningxin) [reply examples/lenet/index.html:104]: good catch. I'll fix it. Thanks!

Comment (huningxin) [reply examples/lenet/index.html]: Fixed in the latest commits. Please take another look.

**Comment by BruceDai** [comment]
@huningxin LGTM, thanks.

**Comment by huningxin** [comment]
Thanks review and merging.

@ibelem , could you please help publish the example to https://intel.github.io/webml-polyfill/examples/lenet. The weights can be downloaded by referring to https://github.com/intel/webml-polyfill/tree/master/examples/lenet#download-model. Thanks!

**Comment by ibelem** [comment]
The model has been uploaded to AWS.
https://webnnmodel.s3-us-west-2.amazonaws.com/lenet/model/lenet.bin 
@BruceDai @huningxin It's been published to gh-pages branch, please let me know if any issues.

**Comment by huningxin** [comment]
The example works perfectly for me. Thanks @belem !

I can refer it by its URL directly for WebML CG discussion. I don't see the need to add entry into home page currently.

**Pull request merged**